evap/staff/templates/staff_course_form.html

```diff
@@ -64,10 +68,18 @@
                 {% if form.instance.state == "inEvaluation" %}
```

Comment:
please add an if around the well div. it probably will be quite long, but an empty div makes no sense.